api/src/opentrons/protocol_engine/state/labware.py

```diff
@@ -320,6 +320,21 @@ def get_well_definition(
                 f"{well_name} does not exist in {labware_id}."
             ) from e
 
+    def get_well_size(
+        self, labware_id: str, well_name: Optional[str] = None
```

Comment:
Let's not allow `well_name` to be optional unless we actually use/need it

```suggestion
        self, labware_id: str, well_name: str
```